share/completions/exif.fish

```diff
@@ -0,0 +1,89 @@
+function __get_target_files
+  set param_args (string split ', ' -- (string trim (string replace -fr "=\w+.*" '' -- (exif --help))))
+  set tokens (commandline -po)[2..]
+  set current_token 1
+  while test "$current_token" -le (count $tokens)
+    if contains -- $tokens[$current_token] $param_args
+      set -e tokens[$current_token]
+      set -e tokens[$current_token]
+    else if string match -- '-*' $tokens[$current_token]
```

Comment:
This removes all options - can this just be `string match -v '-*'` directly on the `commandline` above?

Reply:
I did it this way so that I can remove the tokens following options that take an additional argument, such as the `--output` option which takes an extra path. I just realized though, that this implementation also has another issue: that if a user enters a path to an image (or other file) that exif can't work with, a visible error will be printed, so I think it's probably fine to go with your method then just ignore errors from exif.